pkg/kgo/hooks.go

```diff
@@ -134,6 +134,9 @@ type BrokerE2E struct {
 	WriteErr error
 	// ReadErr is any error encountered during reading.
 	ReadErr error
+
+	// ClientID is pointer to ID of the client that made the request
```

Comment:
I suspect you originally had this a pointer (this doc and other docs can be updated)